tracer/test/Datadog.Trace.TestHelpers/SpanMetadataV0Rules.cs

```diff
@@ -526,6 +526,24 @@ public static Result IsRabbitMQV0(this MockSpan span) => Result.FromSpan(span)
                 .Matches("component", "RabbitMQ")
                 .IsPresent("span.kind"));
 
+        public static Result IsRemotingClientV0(this MockSpan span) => Result.FromSpan(span)
+             .Properties(s => s
+                .Matches(Name, "remoting.message"))
```

Comment:
After all the OTel API operation naming, I think we should make the following changes that apply both v0 and v1:
- Client OperationName: `"dotnet_remoting.client.request"`
- Server OperationName: `"dotnet_remoting.server.request"`

Reply:
Fixed in a9f073d07f7f3d52b900299a6eb0d9d2a8e9a29b